core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1061,6 +1061,15 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val SHUFFLE_FETCH_CONTINUOUS_BLOCKS_IN_BATCH =
+    ConfigBuilder("spark.shuffle.fetchContinuousBlocksInBatch")
+      .doc("Whether to fetch the continuous blocks in batch. Instead of fetching blocks " +
```

Comment:
nit: `continuous blocks` -> `continuous shuffle blocks`

Reply:
Thanks, done in aa603a2.